tests/test_client.py

```diff
@@ -1,11 +1,23 @@
 from faunadb.client import FaunaClient
 from faunadb.errors import UnexpectedError
+from faunadb import query
 from tests.helpers import FaunaTestCase
 
 class ClientTest(FaunaTestCase):
 
   def test_ping(self):
+    old_time = self.client.last_txn_time.get_time()
     self.assertEqual(self.client.ping("node"), "Scope node is OK")
+    new_time = self.client.last_txn_time.get_time()
+
+    self.assertEqual(old_time, new_time) # client.ping should not update last-txn-time
+
+  def test_client(self):
```

Comment:
Maybe `test_track_last_txn_time` would be a more descriptive name for this test case?